quadriga/src/main/java/edu/asu/diging/quadriga/api/v1/AddNetworkApiController.java

```diff
@@ -43,8 +49,18 @@
      * @return
      */
     @ResponseBody
-    @RequestMapping(value = "/api/v1/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-    public HttpStatus processJson(@RequestBody Quadruple quadruple) {
+    @RequestMapping(value = "/api/v1/collection/{collectionId}/network/add", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
+    public HttpStatus processJson(@RequestBody Quadruple quadruple, @PathVariable String collectionId) {
+        
+        if(collectionId == null || collectionId.trim().isEmpty()) {
```

Comment:
Have you tried this? I'm fairly certain this couldn't happen as the url pattern wouldn't match this case, but I might be wrong.

Reply:
Yes this is not required, we just get NotFound because the pattern doesn't match
Will remove this